src/Main.java

```diff
@@ -1,5 +1,154 @@
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                actionSetter(0,1,buttons,statusBar,model);
+            }
+        });
+
+        buttons[0][2].addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                actionSetter(0,2,buttons,statusBar,model);
+            }
+        });
+
+        buttons[1][0].addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                actionSetter(1,0,buttons,statusBar,model);
+            }
+        });
+
+        buttons[1][1].addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                actionSetter(1,1,buttons,statusBar,model);
+            }
+        });
+
+        buttons[1][2].addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                actionSetter(1,2,buttons,statusBar,model);
+            }
+        });
+
+        buttons[2][0].addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                actionSetter(2,0,buttons,statusBar,model);
+            }
+        });
+
+        buttons[2][1].addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                actionSetter(2,1,buttons,statusBar,model);
+            }
+        });
+
+        buttons[2][2].addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                actionSetter(2,2,buttons,statusBar,model);
+            }
+        });
+
+        clearBoardButton.addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                for (int k = 0; k < 3; k++) {
+                    for (int l = 0; l < 3; l++) {
+                        buttons[k][l].setText("");
+                    }
+                }
+                model.newGame();
+                JOptionPane.showMessageDialog(null, "Click OK to Start", "New Game", JOptionPane.INFORMATION_MESSAGE);
+                statusBar.setText(model.getCurrentPlayer() + "'s turn");
+            }
+        });
+
     }
-}
\ No newline at end of file
+    public static void actionSetter(int i, int j, JButton[][] buttons, JLabel statusBar, TicTacToeModel model) {
+
+                if (model.spotIsEmpty(i, j)) {
+                    model.storeMove(i, j);
+                    buttons[i][j].setText(String.valueOf(model.getCurrentPlayer()));
+                } else {
+                    JOptionPane.showMessageDialog(null, "That spot is taken\nPick a different spot", "Oops...", JOptionPane.INFORMATION_MESSAGE);
+                    //System.exit(0);
+                }
```

Comment:
using static method makes you pass in all the arguments...with a class with all those variable as *instance* variables, it would save you tons of work....and clean up your code